src/components/Auth/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,32 @@
+import { useContext } from 'react';
+import { TodoContext } from '../../../context/TodoContext';
+import { TodoRender } from '../TodoRender/TodoRender';
+
+export const TodoList = () => {
+  const {
+    filtredTodos,
+    setInputValue,
+    inputValue,
+    handleStatusChange,
+    selectedTodoId,
+    setSelectedTodoId,
+    handleChangeTitle,
```

Comment:
do you need all these values here? Can’t you get it from the context on the TodoRender component?